README.md

```diff
@@ -1 +1,9 @@
-This repository is for an upcoming project to provid tools for interfacing with NASA metadata systems.
+# EO Metadata Tools #
+
+This repository is for an upcoming project to provid tools for interfacing with
+NASA metadata systems. Each directory possibly written in different programing
```

Comment:
misspelling....should be programming

Reply:
my spell checked does not agree. provid however was wrong and I fixed that